docs/server/push-duty.rst

```diff
@@ -167,9 +185,20 @@ The tag is pushed to production by ops (wezhou), once approved by QA (Krupa), on
 It is the responsibility of the push hero to follow-up with QA and ops,
 and be around during the push for any unexpected issues.
 
+Extension Workshop
+++++++++++++++++++
+
+We push to extension-workshop on prod manually by creating a pushing a tag without
+the ``--stage`` suffix. For example::
+
+    $ git checkout 2022.02.10-stage
+    $ git tag 2022.02.10
+    $ git push upstream 2022.02.10
+
+You should manually verify Extension Workshop on prod after the push has completed.
```

Comment:
```suggestion
You should manually verify Extension Workshop on prod after the push has completed. Visit https://extensionworkshop.com and view any pages that have been changed since the last push to verify they exist and are rendering properly.
```